spec/RangedSetTest.ts

```diff
@@ -154,6 +154,18 @@ describe('RangedSet: ', () => {
         expect(union.isEquals(expected)).toBe(true);
     });
 
+   it("should perform proper inside and contains test ", () => {
+        let oneToTen     = new RangedSet(new IPv4("0.0.0.1"), new IPv4("0.0.0.10"));
+        let fiveToFifteen = new RangedSet(new IPv4("0.0.0.5"), new IPv4("0.0.0.15"));
+        let zeroToTwenty  = new RangedSet(new IPv4("0.0.0.0"), new IPv4("0.0.0.20"));
+        expect(oneToTen.contains(fiveToFifteen)).toBe(false);
+        expect(oneToTen.inside(fiveToFifteen)).toBe(false);
+
+        expect(oneToTen.inside(zeroToTwenty)).toBe(true);
+        expect(zeroToTwenty.inside(oneToTen)).toBe(true);
+
```

Comment:
Also can you be kind enough to split the assertion to individual tests? It makes it easier to know which particular assertion is failing when running the tests. Thanks!